packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,778 @@
+
+<ApiOperation method="get" url="/policies/${policyId}" />
+
+Given a `policyId` identifier, returns a [Policy Object](#okta-identity-engine-policy-objects).
+
+#### Request Body
+
+| Property | Type   | Description                          |
+|----------|--------|--------------------------------------|
+| policyId | String | The unique identifier of the Policy. |
+
+#### Request Query Parameters
+
+None
+
+#### Response Body
+
+A [Policy Object](#okta-identity-engine-policy-objects).
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/${policyId}"
+```
+
+##### Response
+
+```json
+{
+    "id": "rs10xzYOyK9ux6v70g4",
+    "name": "Default Policy",
+    "type": "Okta:IdpRouting",
+    "status": "ACTIVE",
+    "default": true,
+    "_links": {
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10xzYOyK9ux6v70g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10xzYOyK9ux6v70g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Syntax for All Other Operations
+
+See <https://developer.okta.com/docs/reference/api/policy/#policy-api-operations> for request syntax for all other operations.
+
+## Okta Identity Engine Policy Objects
+
+Okta Identity Engine introduces a set of new Policy and Rule objects that are different from the existing Policy and API objects:
+
+* [IdP Routing Policy Object](#idp-routing-policy-object)
+* [IdP Routing Rule Object](#idp-routing-rule-object)
+* [Identifier Match Policy Object](#identifier-match-policy-object)
+* [Identifier Match Rule Object](#identifier-match-rule-object)
+* [Unknown User Policy Object](#uknown-user-policy-object)
+* [Unknown User Rule Object](#unknown-user-rule-object)
+* [Sign On Policy Object](#sign-on-policy-object)
+* [Sign On Rule Object](#sign-on-rule-object)
+* [User Profile Policy Object](#user-profile-policy-object)
+* [User Profile Rule Object](#user-profile-rule-object)
+
+Default instances of each Policy type are created automatically when Okta Identity Engine is enabled for your org. Each of the default Policy objects also has a default Rule that is created automatically.
+
+### IdP Routing Policy Object
+
+The IdP Routing Policy Object determines which IdP users are routed to. One IdP Routing Policy object is created by default. Additional IdP Routing Policy objects cannot be created. Currently, the only IdP you can configure for use with the Okta Identity Engine Pipeline is the Okta IdP, so that, although this object needs to exist, it cannot change the behavior of the pipeline.
```

Comment:
We should be consistent on the casing of `object`

Reply:
Changed to lowercase throughout when not part of a section heading (or reference to a section heading.